src/LogicTasks/Semantics/Pick.hs

```diff
@@ -43,13 +54,13 @@ description PickInst{..} = do
       translate $ do
         german "Betrachten Sie die folgende Formel:"
         english "Consider the following formula:"
-      indent $ code $ availableLetter (literals sTable) : " = " ++ show sTable
+      indent $ code $ availableLetter (literals sTable) : " = " ++ concatMap display trees
```

Comment:
Is this tested? I have a hard time believing that it creates the correct output. Doesn't it offer all the formulas now, instead of just the relevant one?

Reply:
I fixed it